src/test/scala/uk/gov/hmrc/agentmtdidentifiers/model/EnrolmentKeySpec.scala

```diff
@@ -45,6 +45,12 @@ class EnrolmentKeySpec extends AnyFlatSpec with Matchers {
     EnrolmentKey.deconstruct("HMRC-PT~NINO~someId") shouldBe (("HMRC-PT", "someId"))
     an[Exception] shouldBe thrownBy(EnrolmentKey.deconstruct("HMRC-FAKE-SVC~NINO~AB123456Z"))
   }
+  it should "build enrolment keys from enrolments correctly" in {
+    val vatEnrolment = Enrolment("HMRC-MTD-VAT", "Activated", "Joe", Seq(Identifier("VRN", "123456789")))
+    EnrolmentKey.fromEnrolment(vatEnrolment) shouldBe "HMRC-MTD-VAT~VRN~123456789"
+    val cbcEnrolment = Enrolment("HMRC-CBC-ORG", "Activated", "Joe", Seq(Identifier("URN", "01010101"), Identifier("cbcId", "XACBC0123456789")))
```

Comment:
didn't you mean UTR here?

Reply:
Yes, you're right, good spot! Not that it makes any difference since it's just a test and doesn't do any validation but I'll correct it